addon/components/utils/product-row.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="fx-row fx-gap-px-18 fx-xalign-center padding-right-px-18 product-row {{if @plain 'product-row--plain'}}"
      ...attributes>
   <div class="product-image-container {{if this.isDefaultImg 'product-image-container--default'}}">
-    <img src={{this.productImageUrl}} onerror={{this.imageLoadError}} />
+    <img src={{this.productImageUrl}} onerror={{this.imageLoadError}} alt="Product" />
```

Comment:
you can have `alt="" role="presentation"` instead, so the linter will leave alone without providing an `alt`